.ci/Jenkinsfile

```diff
@@ -2,7 +2,7 @@ pCloud = "colo-research-01"
 gitUrl = null
 gitBranch = null
 gitCommit = null
-pTimeout = '1800' // in seconds
+pTimeout = '3600' // in seconds
```

Comment:
Is this needed because the `pytorch_mofed` image build actually takes longer or did you just do it because you were combating the test timeouts?

Reply:
Can the higher timeout be docker-image specific rather than global? Would still like pytests to be 30 minutes max.

Reply:
I'm fine with undoing this change. Jenkins was just highly sus yesterday; I don't think this is still needed.